### ISSUE TITLE
Make UI For above bash script

### ISSUE THREAD

**Issue by stellaharmony** [mbhole1857/Bash_Login_Panel_DBA#1]
Develop a simple registration/login portal front end.
You can refer to the repository for more.
You can choose any language or framework for front-end

**Comment by ajay-droid** [comment]
I would like to work on this. Could you assign it to me? @mbhole1857

**Comment by ajay-droid** [comment]
Can I implement the registration / login portal using html ,css and javascript? @mbhole1857

**Comment by stellaharmony** [comment]
> Can I implement the registration / login portal using html ,css and javascript? @mbhole1857

Yes, you can I have already written the backend script you just have to make a UI for the same.

**Comment by RawPlutonium** [comment]
Is the backend script the main backend or can I create a new script with another language/framework?

**Comment by stellaharmony** [comment]
@RawPlutonium  Bash script is written for backend you have to figure out how you can integrate it with the front-end.
 P.S: Script is created for Bash-DBA Connection Learning Purpose.
Let me know if you have any questions.

**Comment by RawPlutonium** [comment]
I'm getting a bug -  `./account.sh: 45: Syntax error: "elif" unexpected (expecting "then")`
initially before deleting the definition for bash: `bash: ./account.sh: /usr/bin/bash^M: bad interpreter: No such file or directory`
Was this created on a Windows environment?
### Edit
Fixed it on my linux computer by running : `sed -i -e 's/\r$//' account.sh`
So I suppose anyone running this on linux should do the above then : `chmod +x account.sh`
Followed by run : `./account.sh`

**Comment by RawPlutonium** [comment]
Another question is, is the database `feedbyme` supposed to be created explicitly? Like `CREATE DATABASE feedbyme` so as to allow the queries to run successfully?  And as well as to run the sql file on program start? i.e : `mysql -u {username} -p {password}  < feedbyme.sql` to allow for the former to be done and also for the sql file to be run?

**Comment by stellaharmony** [comment]
@RawPlutonium not necessary we already have one file called feedbyme.sql.and yes it will work the same way you've mentioned
`mysql -u {username} -p {password} < feedbyme.sql`
